app/webpacker/components/identity-verification-form.js

```diff
@@ -0,0 +1,38 @@
+class IdentityVerificationForm {
```

Comment:
```suggestion
class NameInitialsForm {
```
?

On s’en sert pour vérifier l’identification, mais le composant en lui-même ne s’occupe que de demander trois lettres.